atlasdb-config/src/main/java/com/palantir/atlasdb/factory/TransactionManagers.java

```diff
@@ -346,12 +438,12 @@ private static void initializeSweepEndpointAndBackgroundProcess(
     }
 
     private static SpecificTableSweeper initializeSweepEndpoint(
-            Environment env,
+            Consumer<Object> env,
             KeyValueService kvs,
             SerializableTransactionManager transactionManager,
             SweepTaskRunner sweepRunner,
             BackgroundSweeperPerformanceLogger sweepPerfLogger,
-            Supplier<SweepBatchConfig> sweepBatchConfig,
+            com.google.common.base.Supplier<SweepBatchConfig> sweepBatchConfig,
```

Comment:
ick. can these just be java suppliers?

Reply:
this wasn't a change; you'll notice I swapped the import from guava to java to prevent un-suspecting contributors such as myself from adding more guava things without realizing. We can move these to java in a separate PR